data/presets_default.json

```diff
@@ -485,42 +501,29 @@
       "logic_lens_shadow",
       "logic_lens_spirit"
     ],
-    "starting_equipment": [
+    "logic_earliest_adult_trade":              "claim_check",
+    "logic_latest_adult_trade":                "claim_check",
+    "starting_equipment":                      [
       "deku_shield"
     ],
-    "starting_items": [
+    "starting_items":                          [
       "ocarina",
       "lens"
     ],
-    "starting_songs": [],
-    "start_with_consumables": true,
-    "start_with_rupees": false,
-    "starting_hearts": 3,
-    "skip_child_zelda": true,
-    "no_escape_sequence": true,
-    "no_guard_stealth": true,
-    "no_epona_race": true,
-    "skip_some_minigame_phases": true,
-    "complete_mask_quest": false,
-    "useful_cutscenes": false,
-    "fast_chests": true,
-    "free_scarecrow": false,
-    "fast_bunny_hood": true,
-    "plant_beans": false,
-    "chicken_count_random": false,
-    "chicken_count": 7,
-    "big_poe_count_random": false,
-    "big_poe_count": 1,
-    "ocarina_songs": "off",
-    "correct_chest_appearances": "off",
-    "clearer_hints": true,
-    "no_collectible_hearts": false,
-    "hints": "always",
+    "starting_songs":                          [
+      "prelude"
+    ],
+    "ocarina_songs":                           false,
+    "correct_chest_sizes":                     false,
```

Comment:
These settings seem to have been generated on stable. They need to be updated for settings changes in Dev, such as `ocarina_songs` having multiple options and `correct_chest_sizes` having been replaced with `correct_chest_appearances`.

Reply:
Hmm i'll take a look at these, this was literally just edited directly in the file on dev, I didn't generate it on stable.   So that might have already been tehre.